libraries/helpers.rb

```diff
@@ -430,6 +430,7 @@ def setup_munge_head_node
       # Enforce correct permission on the key
       chmod 0600 /etc/munge/munge.key
     HEAD_CREATE_MUNGE_KEY
+    creates '/etc/munge/munge.key'
```

Comment:
what's the motivation for adding this? from what i understand the `creates` directive prevents the resource from running again if the file already exists. is there a reason we would expect anything under /etc to have been already generated by a customer?

Reply:
**Reason for the change**
The munge key is generated in the head node and then copied into the head node home to be shared with computed nodes. So, assuming we should not overwrite existing the content in /home, then we should not re-generate the key in /etc.

**Alternative approach (out of scope for this PR)**
Use a directory different from /home to share data between head node and compute nodes.